meta/Meta.cpp

```diff
@@ -7594,6 +7594,52 @@ void Meta::meta_sai_on_fdb_event(
     }
 }
 
+void Meta::meta_sai_on_nat_event_single(
+        _In_ const sai_nat_event_notification_data_t& data)
+{
+    SWSS_LOG_ENTER();
+
+    const sai_object_meta_key_t meta_key_nat = { .objecttype = SAI_OBJECT_TYPE_NAT_ENTRY, .objectkey = { .key = { .nat_entry = data.nat_entry } } };
+
+    switch (data.event_type)
+    {
+        case SAI_NAT_EVENT_AGED:
+
```

Comment:
should nat entry be removed here when aged event was received ? similar like in fdb entry aged ?

Reply:
Hi Kamil, thanks for reviewing.
Yes, nat entry gets removed when aged event is processed in SWSS. Natorch triggers deletion of the kernel conntrack entry for the cleanup. This is similar to the existing case where platforms that do not have NAT aging will rely on hitbit timer for cleanup.

Reply:
yes, but i mean should it be removed here from Meta database, since if it will be removed automatically then meta database will have inconsistent state with actual device

Reply:
from your pictures seems like nat entry will be deleted after event explicitly by natorch, so no code will be required here, but i would prefer to put at least comment in this part of code why there is no explicit delete

this behavior then is different than fdb, so SAI is just generating NAT aged event but entry is still there preserved ? then this would need to be explicitly added in sai documentation repo so all vendors would have the same view on this

Reply:
Added comments